crates/bevy_asset/src/handle.rs

```diff
@@ -15,7 +15,7 @@ use bevy_utils::Uuid;
 use crossbeam_channel::{Receiver, Sender};
 use serde::{Deserialize, Serialize};
 
-/// A unique, stable asset id
+/// An unique, stable asset id.
```

Comment:
```suggestion
/// A unique, stable asset id.
```
Sorry, English is dumb. "unique" is pronounced "yoo-neek", which doesn't need the liaison of "an".

Reply:
I've spent quite some time before making that change, but I should've googled it to be sure. Thanks!